mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -214,16 +213,16 @@ def test_brainvision_data_lowpass_filters():
 
     assert_true(all(any([lp, hp]) for lp, hp in expected_warnings))
 
-    assert_equal(raw.info['highpass'], 0.1)
+    assert_equal(raw.info['highpass'], 1. / (2 * np.pi * 10))
     assert_equal(raw.info['lowpass'], 250.)
 
     # Homogeneous lowpass in seconds
     raw = _test_raw_reader(
         read_raw_brainvision, vhdr_fname=vhdr_lowpass_s_path,
         montage=montage, eog=eog, event_id=event_id)
 
-    assert_equal(raw.info['highpass'], 0.1)
-    assert_equal(raw.info['lowpass'], 250.)
+    assert_equal(raw.info['highpass'], 1. / (2 * np.pi * 10))
+    assert_equal(raw.info['lowpass'], 1. / (2 * np.pi * 0.004))
```

Comment:
Leave the "Hertz" obvious, see comment below

Reply:
see above